shoop/admin/modules/orders/static_src/create/reducers/customer.js

```diff
@@ -31,7 +31,13 @@ function setAddressProperty(state, {payload}) {
 }
 
 export default handleActions({
-    setCustomer: ((state, {payload}) => payload),
+    setCustomer: ((state, {payload}) => _.assign(state, {
```

Comment:
If those keys aren't there in the payload, they won't be cleared.

![image](https://cloud.githubusercontent.com/assets/58669/11495828/db8eb52c-9816-11e5-928d-1e8f4e497d14.png)